management-api-server/src/main/java/com/datastax/mgmtapi/resources/NodeOpsResources.java

```diff
@@ -663,6 +663,37 @@ public Response searchIndexRebuild(
         });
   }
 
+  @POST
+  @Path("/reload-truststore")
```

Comment:
discussion/suggestion/question/<I'm not sure if there's a correct type annotation>: The use of "-" here is a bit weird compared to the rest of the methods in mgmt-api. For similar reloads we already "schema/reload", "seeds/reload" vs. "reload-truststore". Could we instead have "/truststore/reload" ?

Reply:
I've made some changes to allow for the fact that we may one day have an endpoint to do things with client encryption too. As a result I've also adopted something similar to this.